orchestrator/dags/general_data_import.py

```diff
@@ -74,19 +74,18 @@ def extract_timetable_lines(ti) -> None:
         for line in file:
             if "Linia:" in line and len(line.split()[1]) < 3 and line.split()[1].isdecimal():
                 tram_number = line.split()[1]
-                f = open(f'{out_dir}tram_line{tram_number}.txt', "w", encoding="utf8")
-                while True:
-                    try:
-                        f.write(line)
-                        line = next(file)
-                    except StopIteration:
-                        # there is no lines left
-                        break
-                    if '#WK' in line:
-                        # we've reached the end of the data for given tram line
-                        f.close()
-                        break
-                continue    
+                with open(f'{out_dir}tram_line{tram_number}.txt', "w", encoding="utf8") as f:
```

Comment:
Would be nice to rename variables assigned to opened files, e.g. `infile` instead `file` and `outfile` insted `f` :slightly_smiling_face: